specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerApps.json

```diff
@@ -755,6 +755,29 @@
                 "modelAsString": true
               }
             },
+            "runningStatus": {
+              "description": "Running status of the Container App.",
+              "enum": [
+                "Progressing",
+                "Running",
+                "Stopped",
+                "Suspended",
+                "Ready"
+              ],
+              "x-enum-descriptions": [
+                "Container App is transitioning between Stopped and Running states.",
+                "Container App is in Running state",
+                "Container App is in Stopped state",
+                "Container App Job is in Suspended state",
+                "Container App Job is in Ready state"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "ContainerAppRunningStatus",
+                "modelAsString": true
+              }
```

Comment:
Add as 

"x-ms-enum": {
        "name": "ContainerAppRunningStatus",
        "modelAsString": true,
        "values": [
          {
            "value": "Progressing",
            "description": "<<<Meaning of it>>>."
          },
          {
            "value": "Running",
            "description": "<<<Meaning of it>>>"
          },
....... <<<< and others >>>>
        ]
      }